setup.py

```diff
@@ -31,6 +33,99 @@
         version = match.group(1)
 
 
+def _get_help_output(topic, lines=False):
+    TRANSLATE = {
+        'tar-content': 'tar_content',
+    }
```

Comment:
Does this apply to all the topics with a dash? If so, you could just replace the dash.